Dockerfile

```diff
@@ -1,12 +1,9 @@
 FROM python:3.4.8-jessie
-
 WORKDIR /usr/src/app
+EXPOSE 8080
```

Comment:
You are leaving yourself exposed! Cover your butt

Reply:
🚽